### DISCUSSION TITLE
Trivy Scan Error

### DISCUSSION THREAD

**Discussion by Susanta2000** [aquasecurity/trivy#4104] [Q&A] [answered]
Hi there,
  I am using Trivy  for vulnerabilities and misconfigurations in images. I want to scan my aws account's resource. But when I am running $trivy aws command I am getting following error:
2023-04-21T19:01:04.426+0530	FATAL	scan error: unable to initialize a scanner: unable to initialize a docker scanner: 3 errors occurred:
	* unable to inspect the image (index.docker.io/library/aws:latest): Error: No such image: index.docker.io/library/aws:latest
	* unable to initialize Podman client: no podman socket found: stat /run/user/1000/podman/podman.sock: no such file or directory
	* GET https://index.docker.io/v2/library/aws/manifests/latest: UNAUTHORIZED: authentication required; [map[Action:pull Class: Name:library/aws Type:repository]]

**Comment by Susanta2000** [comment]
Can anyone help me?

**Reply by knqyf263** [reply, marked as answer]
Please refer to this page if you want to scan container images.
https://aquasecurity.github.io/trivy/v0.40/docs/target/container_image/
